docs/source/docs/data-formats/supported_formats.md

```diff
@@ -130,6 +130,10 @@ List of supported formats:
   - [Format specification](http://host.robots.ox.ac.uk/pascal/VOC/voc2012/htmldoc/index.html)
   - [Dataset example](https://github.com/openvinotoolkit/datumaro/tree/develop/tests/assets/voc_dataset)
   - [Format documentation](../explanation/formats/pascal_voc)
+- Segment Anything (a.k.a SA-1B) (`segmentation`)
```

Comment:
This may be `detection`, `segmentation`. Am I right?

Reply:
Fixed in https://github.com/openvinotoolkit/datumaro/pull/1009/commits/652cd5d8027f8cef4c1545191bcc47dcae9278d0